src/webpki.rs

```diff
@@ -140,6 +140,13 @@ impl<'a> EndEntityCert<'a> {
         })
     }
 
+    /// Retrieves the raw extensions that were found in the certificate but were
+    /// not recognized by the webpki library. These could be custom extensions
+    /// or standard extensions not relevant to how the library operates
+    pub fn unrecognized_extensions(&self) -> std::collections::HashMap<&'a[u8], untrusted::Input<'a>> {
```

Comment:
This can be marked as `#[inline]`.  Also (and this is a matter of style, so feel free to ignore) I tend to return borrowed values from these sorts of getter functions: that way if the caller can get away with using the borrowed value (e.g. they only need to read from it) then they can avoid any allocation.

Reply:
Good point. I cloned to avoid any ownership problems that often come up with hash maps (they are particularly sticky for ownership). I should probably see if a borrow works.